.travis.yml

```diff
@@ -1,23 +1,10 @@
-#This file is generated by ModuleSync, do not edit.
----
-sudo: false
 language: ruby
-cache: bundler
-script: "bundle exec rake validate lint spec"
-matrix:
-  fast_finish: true
-  include:
-  - rvm: 2.1.6
-    bundler_args: --without system_tests
-    env: PUPPET_GEM_VERSION="~> 4.0" STRICT_VARIABLES="yes"
-  - rvm: 2.1.5
-    bundler_args: --without system_tests
-    env: PUPPET_GEM_VERSION="~> 3.0" FUTURE_PARSER="yes"
-  - rvm: 2.1.5
-    bundler_args: --without system_tests
-    env: PUPPET_GEM_VERSION="~> 3.0"
-  - rvm: 1.9.3
-    bundler_args: --without system_tests
-    env: PUPPET_GEM_VERSION="~> 3.0"
+rvm: 2.1.9
```

Comment:
please don't make changes to our .travis.yml file. This file is templated from out site.